### ISSUE TITLE
Issues with Solr7 + 8 because of openjdk-11 on Ubuntu 18.04

### ISSUE THREAD

**Issue by ayalon** [geerlingguy/drupal-vm#1940]
### Issue Type
 - Bug Report / Support Request

### Your OS

 - **macOS** (version)
 - **Windows** (version)
 - **Linux** (version)

### Summary
I had huge troubles with Solr 7 and Solr 8 when using the latest VM box with ubuntu 18.04.
After a long journey, I finally figured out the main cause. Without any further configuration, automatically openjdk-11 is installed, if java is enabled. 

I downgraded to openjdk-8:
```
java_packages:
  - openjdk-8-jdk
```

Basically you can get Solr up and running, but as soon as you do complex stuff like indexing oder deleting the index, I constantly lost the connection to the solr server.

Maybe this helps someone else to get in the right direction.
